generators/app/templates/server/.env

```diff
@@ -0,0 +1,12 @@
+# PORT
+PORT=4000
```

Comment:
.env should be in `.gitignore` and only add `.env.example` to repo .... ya amateur